docs/gen_ref_pages.py

```diff
@@ -8,17 +8,22 @@
 import mkdocs_gen_files
 
 
+if not Path('src/easybuild-framework/easybuild').is_dir():
+    print("WARNING  -  EasyBuild Framework is not available. API documentation will not be generated.")
+    exit(0)
+
+
 # remove the problematic py2.py and symlink the py3.py in its place
-Path("src/easybuild/tools/py2vs3/py2.py").unlink(missing_ok=True)
-Path("src/easybuild/tools/py2vs3/py2.py").symlink_to(Path("py3.py"))
+Path("src/easybuild-framework/easybuild/tools/py2vs3/py2.py").unlink(missing_ok=True)
+Path("src/easybuild-framework/easybuild/tools/py2vs3/py2.py").symlink_to(Path("py3.py"))
```

Comment:
Avoid repeating the "src/easybuild-framework", introduce a constant on top of the script?

Reply:
done in ee556fc